### PR TITLE
Unit Tests: fix all PHPCS errors

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* No changes in functionality, just fixing PHPCS errors.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Not much to test here. Do the tests pass?

### CODE REVIEW

**Review by sdixon194 — CHANGES_REQUESTED** [review]
Looks god! Just two small things.

File: projects/plugins/jetpack/tests/php/bootstrap.php [hunk 1]
```diff
@@ -76,7 +76,7 @@
 	echo "Disregard Core's -c tests/phpunit/multisite.xml notice below." . PHP_EOL;
 }
 
-if ( '1' != getenv( 'JETPACK_TEST_WOOCOMMERCE' ) ) {
+if ( '1' != getenv( 'JETPACK_TEST_WOOCOMMERCE' ) ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseNotEqual
```
[the remark this answers is not in the document]
Comment (jeherve) [reply projects/plugins/jetpack/tests/php/bootstrap.php]: Yeah, you're right. Fixed in f4b6cb64d86c64b09409c77a4f7dac96b02d6d97

**Review by sdixon194 — DISMISSED** [review]

**Comment by sdixon194** [comment]
Ready to go! Just will need a master/merge after the release.

**Comment by jeherve** [comment]
@sdixon194 I had to fix merge conflicts so your review was dismissed. Do you think you could approve again? 

Thank you!

**Review by sdixon194 — APPROVED** [review]

**Pull request merged**